go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	filippo.io/cpace v0.0.0-20200503185815-340c58da85ed
 	github.com/NYTimes/gziphandler v1.1.1
 	github.com/pion/webrtc/v2 v2.2.11
-	golang.org/x/crypto v0.0.0-20200323165209-0ec3e9974c59
+	golang.org/x/crypto v0.0.0-20200602180216-279210d13fed
```

Comment:
is this commit intentional?

Reply:
Sorry my fault, this was just for a test, it should not go into the MR